syft/frameworks/mpc/crypten/__init__.py

```diff
@@ -0,0 +1,15 @@
+import crypten
```

Comment:
Those are needed to have a simple method to put (key, value) objects in the store